primer/primer.ru.robomarkup

```diff
@@ -364,24 +364,24 @@ fun sum(list[])
 seq nums[] = 1, 2, 3
 var total = sum(nums[])
 
-total is 6
+# total будет равен 6
```

Comment:
This should be left as is, I think. `total is 6` isn't a comment, when this example runs the extra line allows the inline debug for `var total = sum(nums[])` to show up better.